editor/src/core/es-modules/package-manager/module-resolution.ts

```diff
@@ -183,22 +209,31 @@ interface PartialPackageJsonDefinition {
   name?: string
   main?: string
   module?: string
+  browser?: string | MapLike<string | null>
 }
 
 export function parsePartialPackageJsonDefinition(
   value: unknown,
 ): ParseResult<PartialPackageJsonDefinition> {
-  return applicative3Either(
-    (name, main, module) => {
+  return applicative4Either(
+    (name, main, module, browser) => {
       let result: PartialPackageJsonDefinition = {}
       setOptionalProp(result, 'name', name)
       setOptionalProp(result, 'main', main)
       setOptionalProp(result, 'module', module)
+      setOptionalProp(result, 'browser', browser)
       return result
     },
     optionalObjectKeyParser(parseString, 'name')(value),
     optionalObjectKeyParser(parseString, 'main')(value),
     optionalObjectKeyParser(parseString, 'module')(value),
+    optionalObjectKeyParser(
+      parseAlternative<string | MapLike<string | null>>(
```

Comment:
According to the official spec this should be `parseAlternative<string | MapLike<string | boolean>>`

Reply:
fixed!